### PR TITLE
Provisioning code page screen redesigned to spec

### PR DESCRIPTION
**Pull request by pzduniak**
cc @keybase/y2ksquad 

image
image
image
image
image

The phone icon needs negative side margins to look great here, I opted for the simplest possible solution instead.

### CODE REVIEW

File: shared/provision/code-page/index.tsx [hunk 1]
```diff
@@ -499,9 +520,17 @@ const styles = Styles.styleSheetCreate(
...
       }),
       deviceIcon: {
+        height: 32,
+        width: 32,
+      },
+      deviceIconDesktop: {
         marginLeft: Styles.globalMargins.xtiny,
         marginRight: Styles.globalMargins.xxtiny,
       },
+      deviceIconMobile: {
```
Comment (buoyad) [inline shared/provision/code-page/index.tsx]: seems offset to the right? 
image

File: shared/provision/code-page/index.tsx [hunk 2]
```diff
@@ -374,75 +375,95 @@ const ViewText = (props: Props) => (
...
+                  <Kb.Icon
+                    type="iconfont-nav-2-hamburger"
+                    color={Styles.globalColors.white}
+                    sizeType="Default"
+                    style={styles.hamburger}
+                  />
+                  <Kb.Icon type="iconfont-arrow-right" color={Styles.globalColors.white} sizeType="Tiny" />
+                </>
+              )}
+              <Kb.Text type={textType} style={[styles.instructions]}>
```
Comment (buoyad) [inline shared/provision/code-page/index.tsx]: :fire: `[]`

Comment (pzduniak) [reply shared/provision/code-page/index.tsx]: !

File: shared/provision/code-page/index.tsx (hunk 1 above)
Comment (pzduniak) [reply shared/provision/code-page/index.tsx]: Mobile, post fix:
image
image

Comment (pzduniak) [reply shared/provision/code-page/index.tsx]: image
image

Comment (buoyad) [reply shared/provision/code-page/index.tsx]: Looks good

**Pull request merged**